components/org.wso2.carbon.identity.remotefetch.core/src/main/java/org/wso2/carbon/identity/remotefetch/core/util/RemoteFetchConfigurationUtils.java

```diff
@@ -30,6 +34,27 @@
  */
 public class RemoteFetchConfigurationUtils {
 
+    private static final Log log = LogFactory.getLog(RemoteFetchConfigurationUtils.class);
+
+    private static int defaultItemsPerPage;
```

Comment:
These logic need to be moved to 
RemoteFetchConfigurationServiceImpl.java
while retaining the previous getters.